### DISCUSSION TITLE
Two methods to use --portable by default. Is one better?

### DISCUSSION THREAD

**Discussion by Steve-Z** [go-shiori/shiori#489] [Q&A]
I'm a new user on Pop!_OS 22.04. I am using the `shiori` binary, version 1.5.3. I would like to use `shiori --portable` by default. I can imagine doing this by setting the environment variable `SHIORI_DIR='/path/to/shiori-data'`(same parent dir as binary). OR I can set a bash alias: `shiori='shiori --portable'`. Is one method better than the other? Is there an even better way?

**Comment by fmartingr** [comment]
Hey @Steve-Z, as you said, `--portable` just uses the executable path as data folder for shiori, so using `SHIORI_DIR` or `--portable` doesn't make any difference if you're using the executable folder to store the data. If I had to choose, I would do `SHIORI_DIR`, better be explicit about what you want to do. :)
